.jshintrc

```diff
@@ -29,5 +29,8 @@
     "maxparams": 3,
 
     // Limit the nesting to three levels for callbacks.
-    "maxdepth": 3
+    "maxdepth": 3,
+
+    // Identify possibly unsafe line breaks.
+    "laxbreak": true
```

Comment:
I guess it would be a good thing to migrate all these checks to eslint at some point.